src/site/stages/build/drupal/graphql/nodeCampaignLandingPage.graphql.js

```diff
@@ -507,6 +517,10 @@ module.exports = `
           fieldDescription
           fieldDuration
           fieldMediaVideoEmbedField
+          fieldPublicationDate {
+            date
+            value
+          }
```

Comment:
Will you be making the transformer changes in another ticket?

Reply:
Maybe, we'll see